tests/sentry/receivers/test_onboarding.py

```diff
@@ -482,3 +484,130 @@ def test_onboarding_complete(self):
+                                        ],
+                                        "context_line": "      return fn.apply(this, wrappedArguments);",
+                                        "post_context": [
+                                            "    } catch (ex) {",
+                                            "      ignoreNextOnError();",
+                                            "",
+                                            "      withScope((scope) => {",
+                                            "        scope.addEventProcessor((event) => {",
+                                        ],
+                                        "in_app": False,
+                                        "data": {
+                                            "sourcemap": "https://s1.sentry-cdn.com/_static/dist/sentry/sourcemaps/vendors-node_modules_emotion_is-prop-valid_node_modules_emotion_memoize_dist_memoize_brows..."
+                                        },
+                                    }
+                                ]
+                            },
+                            "raw_stacktrace": {
+                                "frames": [
+                                    {
+                                        "function": "o",
+                                        "filename": "/_static/dist/sentry/chunks/vendors-node_modules_emotion_is-prop-valid_node_modules_emotion_memoize_dist_memoize_browser_-4fe4bd.255071ceadabfb67483c.js",
+                                        "abs_path": "https://s1.sentry-cdn.com/_static/dist/sentry/chunks/vendors-node_modules_emotion_is-prop-valid_node_modules_emotion_memoize_dist_memoize_browser_-4fe4bd.255071ceadabfb67483c.js",
+                                        "lineno": 2,
+                                        "colno": 37098,
+                                        "pre_context": [
+                                            "/*! For license information please see vendors-node_modules_emotion_is-prop-valid_node_modules_emotion_memoize_dist_memoize_browser_-4fe4bd. {snip}"
+                                        ],
+                                        "context_line": "{snip} .apply(this,arguments);const i=o.map((e=>c(e,t)));return e.apply(this,i)}catch(e){throw l(),(0,i.$e)((n=>{n.addEventProcessor((e=>(t.mechani {snip}",
+                                        "post_context": [
+                                            "//# sourceMappingURL=../sourcemaps/vendors-node_modules_emotion_is-prop-valid_node_modules_emotion_memoize_dist_memoize_browser_-4fe4bd.fe32 {snip}"
+                                        ],
+                                        "in_app": False,
+                                    },
+                                ],
+                            },
+                        }
+                    ]
+                },
+            },
+        )
+
+        assert send_robust.called
```

Comment:
@priscilawebdev can you add tests where we do not call `send_robust`? Those tests are honestly more important so we don't generate way too many events.

Reply:
I would also move any test that mocks `send_robust` in the `test_event_manager.py` file since we're not actually testing the implementation of the receiver in these tests.